AuthenticatorShared/Core/Vault/Repositories/AuthenticatorItemRepository.swift

```diff
@@ -89,35 +90,98 @@ protocol AuthenticatorItemRepository: AnyObject {
     private let authenticatorItemService: AuthenticatorItemService
+
+    /// Service to determine if the sync feature flag is turned on.
+    private let configService: ConfigService
+
+    /// Service to encrypt/decrypt locally stored Authenticator items.
     private let cryptographyService: CryptographyService
+
+    /// Error Reporter for any errors encountered
     private let errorReporter: ErrorReporter
+
+    /// Service to fetch items from the shared CoreData store - shared from the main Bitwarden PM app.
+    private let sharedItemService: AuthenticatorBridgeItemService
+
+    /// A protocol wrapping the present time.
     private let timeProvider: TimeProvider
+
+    /// A service for refreshing TOTP codes.
     private let totpService: TOTPService
 
     // MARK: Initialization
 
     /// Initialize a `DefaultAuthenticatorItemRepository`
     ///
     /// - Parameters:
-    ///   - authenticatorItemService
-    ///   - cryptographyService
+    ///   - authenticatorItemService: Service to from which to fetch locally stored Authenticator items.
+    ///   - configService: Service to determine if the sync feature flag is turned on.
+    ///   - cryptographyService: Service to encrypt/decrypt locally stored Authenticator items.
+    ///   - sharedItemService: Service to fetch items from the shared CoreData store - shared from
+    ///     the main Bitwarden PM app.
+    ///   - errorReporter: Error Reporter for any errors encountered
+    ///   - timeProvider: A protocol wrapping the present time.
+    ///   - totpService: A service for refreshing TOTP codes.
     init(
         authenticatorItemService: AuthenticatorItemService,
+        configService: ConfigService,
         cryptographyService: CryptographyService,
         errorReporter: ErrorReporter,
+        sharedItemService: AuthenticatorBridgeItemService,
         timeProvider: TimeProvider,
         totpService: TOTPService
     ) {
         self.authenticatorItemService = authenticatorItemService
+        self.configService = configService
         self.cryptographyService = cryptographyService
         self.errorReporter = errorReporter
         self.timeProvider = timeProvider
         self.totpService = totpService
+        self.sharedItemService = sharedItemService
     }
 
     // MARK: Private Methods
 
+    /// Appends a list of the sections to the item list when sync with the PM app is enabled.
+    ///
+    /// Note: If the `enablePasswordManagerSync` feature flag is turned off, or if the user has not yet
+    /// turned on sync for any accounts, this method simply returns `localSections`.
+    ///
+    /// - Parameters:
+    ///   - localSections: The [ItemListSection] sections for the items locally stored
+    ///   - sharedItems: The shared items that are coming in via sync with the PM app
+    /// - Returns: A list of the sections to display in the item list
+    ///
+    private func appendSyncedItems(
```

Comment:
🎨 Historically, method that started with `append` tended to modify things in place, though that was also as a method on an object itself, rather than as a helper as we here (cf. [Swift API design guidelines](https://www.swift.org/documentation/api-design-guidelines/#name-according-to-side-effects)). I think it might be somewhat more descriptive to call this something like `combinedItems`, so that it describes what it *returns* rather than how it *functions*. It's a small thing in the grand scheme of things, but the name did confuse me at first because of that convention.

(Though I admit we're not the most consistent here, cf. `refreshTotpCodes(on:)`)

Reply:
I changed it to `combinedSections` in the latest commit, because I also noticed that it's returning `[ItemListSection]`, not items (which also might have contributed to the confusion). 👍